components/atoms/ChooseImage.tsx

```diff
@@ -0,0 +1,92 @@
+import { Plus } from '@tamagui/lucide-icons'
+import * as ImagePicker from 'expo-image-picker'
+import React, { useState } from 'react'
+import { Alert, useColorScheme } from 'react-native'
+import { Button, Image, XStack } from 'tamagui'
+
+import getColors from '~/constants/Colors'
+
+const ChooseImage = (): React.ReactElement => {
+  const [selectedImages, setSelectedImages] = useState<string[]>([])
+  const colors = getColors(useColorScheme())
+
+  const pickImages = async (): Promise<void> => {
+    // Yêu cầu quyền truy cập ảnh
+    const permissionResult =
+      await ImagePicker.requestMediaLibraryPermissionsAsync()
+
+    if (!permissionResult.granted) {
+      Alert.alert('Permission to access camera roll is required!')
+      return
+    }
+
+    // Mở trình chọn ảnh với giới hạn 3 ảnh
+    const result = await ImagePicker.launchImageLibraryAsync({
+      allowsMultipleSelection: true,
+      mediaTypes: ImagePicker.MediaTypeOptions.Images, // Cho phép chọn nhiều ảnh
+      selectionLimit: 3 // Giới hạn tối đa 3 ảnh
```

Comment:
Manage the number of photos limit: using Selectionlimit: 3 to limit 3 photos, you can add some logic in Setelectedimages to make sure users cannot add more than the prescribed amount of photos.